docs/CONTRIBUTOR_GUIDE.md

```diff
@@ -0,0 +1,55 @@
+# Contributor Guide
+
+This guide is intended for people who want to start working on `krew` itself.
+If you intend to write a new plugin, see [Developer Guide](./DEVELOPER_GUIDE.md) instead.
+
+## Setting up the environment
+
+Krew is built with go 1.10, but newer versions will do as well.
+Most toolchains will expect that the krew repository is on the `GOPATH`.
+To set it up correctly, do
+```bash
+mkdir -p $(go env GOPATH)/src/sigs.k8s.io/krew
+cd $(go env GOPATH)/src/sigs.k8s.io/krew
+git clone https://github.com/kubernetes-sigs/krew .
+git remote set-url origin --push no_push   # to avoid pushes
+```
+
+## Code style
+
+Krew adheres to standard `golang` code formatting conventions, and also expects imports sorted properly.
+To automatically format code appropriately, run
+```bash
+goimports -w cmd pkg
+```
+
+In addition, a boilerplate license header is expected in all source files.
+
+_All new code should be covered by tests._
+
+
+## Running tests
+
+To run tests locally, the easiest way to get started is with
+```bash
+hack/run-tests.sh
+```
+To run a single tool independently of the other code checks, have a look at the other scripts in [`hack/`](../hack).
+
+## Testing `krew` in a sandbox
+
+After making changes to krew, you should also check that it behaves as expected.
+You can do this without messing up the krew installation on the host system by setting the `KREW_ROOT` environment variable.
```

Comment:
80-char line wraps please :)

Reply:
I usually don't do this for markup languages, because it has one great advantage for me: if I change a sentence, there is exactly one line in the diff.

I'll add the wraps, but please don't make me continue a paragraph on the same line where the previous sentence ended. ;)

Reply:
GitHub handles the visualization of this pretty well. I wouldn’t worry about it. It makes reviewing much easier as well.